src/cli.h

```diff
@@ -0,0 +1,18 @@
+#ifndef YOBEMAG_CLI_H
+#define YOBEMAG_CLI_H
+
+#include <stdint.h>
+
+typedef struct CLIArguments {
+    int logging_level;
```

Comment:
I think this should be `LoggingLevel logging_level`

Reply:
True, did originally not do that, because I used CLIArguements as a parameter in logging.h somewhere, which was unnecessary. Should be fixed now.